pkgs/by-name/sp/spotube/package.nix

```diff
@@ -51,13 +51,13 @@ let
 
     sourceRoot = ".";
 
-    nativeBuildInputs = [ undmg ];
+    nativeBuildInputs = [ undmg wrapGAppsHook ];
```

Comment:
Why is this added ?

Reply:
Ok, but what's the reason to use `wrapGAppsHook` over `makeBinaryWrapper` ?

Reply:
It's a GTK app, not sure if that needs the special wrapping on darwin. But as written I believe this would get double-wrapped, since `dontWrapGApps` would be false by default.

Reply:
> [#297790 (comment)](https://github.com/NixOS/nixpkgs/pull/297790#discussion_r1535277410)

But there was no `wrapGAppsHook` in `nativeBuildInputs` here.